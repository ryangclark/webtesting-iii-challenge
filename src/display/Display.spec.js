// Test away!
import React from 'react';
import { cleanup, render } from 'react-testing-library';
import 'jest-dom/extend-expect';

import Display from './Display';

afterEach(cleanup);

describe('<Display />', () => {
  it('Snapshot test', () => {
    const { container } = render(<Display locked={false} closed={false} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
