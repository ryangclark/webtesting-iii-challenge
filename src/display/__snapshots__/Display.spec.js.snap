// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> Snapshot test 1`] = `
<div
  class="display panel"
>
  <div
    class="led green-led"
  >
    Unlocked
  </div>
  <div
    class="led green-led"
  >
    Open
  </div>
</div>
`;
